src/twisted/conch/test/test_transport.py

```diff
@@ -2242,6 +2242,21 @@ class ClientSSHTransportDHGroupExchangeBaseCase(ClientSSHTransportBaseCase):
     Diffie-Hellman group exchange tests for SSHClientTransport.
     """
 
+    """
+    1536-bit modulus from RFC 3526
+    """
+    P1536 = int(
```

Comment:
[modulos.zip](https://github.com/twisted/twisted/files/5611707/modulos.zip)

There's little to confirm in this diff, but I at least checked that the constant is correct. Attached are my calculations.